packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 17.1.3
```

Comment:
Isn't this a breaking change for Obj-C clients?

Reply:
I assumed it was a bug fix since this creates code that doesn't compile.

Reply:
It generates correctly in other places. At least this was the case for me.

Reply:
Yeah, if it's generating bad references to correctly named declarations, that's just a bugfix.